app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
Or even cleaner might be:

```ruby
module Api
  Config = ::Config::Options.new.tap {|o| o.add_source(YAML_FILE); o.load!}
end
```

then you can call

```ruby
Api::Config.collections
Api::Config.base.name
```

and then support for "reloading" is possible if needed `Api::Config.reload!`